lib/daru/io/importers/redis.rb

```diff
@@ -0,0 +1,66 @@
+require 'daru/io/importers/linkages/redis'
+require 'json'
+
+module Daru
+  module IO
+    module Importers
+      class Redis
+        def initialize(connection={}, *keys)
+          @client = get_client(connection)
+          @keys   = choose_keys(*keys)
+        end
+
+        def load
+          vals = @keys.map { |key| ::JSON.parse(@client.get(key)) }
+
+          case ::JSON.parse(@client.get(@keys[0]))
```

Comment:
There are two HUGE problems with this approach:
1. It is fragile. It assumes that entire key set has exactly the same structure as the first row fetched, which may, or may not, be true in practice. And I believe that the cases when it is not true, the errors would be absolutely cryptic.
2. It has nothing to do with Redis in particular, any importer that receives complicated nested structures, will try to guess how to parse them, again and again.

Let's at least move this "guessing" part to some `Importers::Base` and try to make it a bit more robust (maybe checking _all_ fetched rows instead of the first?..)